src/main/java/uk/gov/hmcts/probate/model/Constants.java

```diff
@@ -30,6 +30,7 @@
     public static final String STATE_GRANT_TYPE_INTESTACY = "SolIntestacyCreated";
     public static final String STATE_GRANT_TYPE_ADMON = "SolAdmonCreated";
     public static final String STATE_GRANT_TYPE_CREATED = "SolAppCreated";
+    public static final String REGEX = "[a-z0-9!#$%&'*+/=?^_`{|}~-]{1,30}(?:\\.[^.\\n]{1,30}){0,30}@[a-z0-9](?:[a-z0-9-.]{0,30}[a-z0-9])?\\.[a-z0-9](?:[a-z0-9-]{0,10}[a-z0-9])?";
```

Comment:
sorry to be pedantic - but can you rename this var please?

Reply:
Done.